cmd/server/assets/codes/bulk-issue.html

```diff
@@ -0,0 +1,302 @@
+{{define "codes/issue-bulk"}}
+
+{{$hasSMSConfig := .hasSMSConfig}}
+
+<!doctype html>
+<html lang="en">
+
+<head>
+  {{template "head" .}}
+  <script
+    src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/4.0.0/crypto-js.min.js"
+    integrity="sha512-nOQuvD9nKirvxDdvQ9OMqe2dgapbPB7vYAMrzJihw5m+aNcf0dX53m6YxM4LgA9u8e9eg9QX+/+mPu8kCNpV2A=="
+    crossorigin="anonymous"></script>
+</head>
+
+<body id="users-import" class="tab-content">
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <div class="card mb-3 shadow-sm">
+      <div class="card-header">Bulk issue</div>
+      <div class="card-body">
+        {{template "beta-notice" .}}
+
+        {{if not $hasSMSConfig}}
+        <div class="alert alert-danger">
+          <span class="oi oi-warning"></span>
+          No SMS provider is configured for this realm.
+          Please contact a realm administrator to enable this feature.
+        </div>
+        {{end}}
+
+        <p>
+          Use this form to bulk issue codes.
+        </p>
+
+        <pre>Example file contents:
+          <code>
+            +12065555555,2020-11-10
+          </code>
+        </pre>
+
+        <p>
+          This form will data and issue a code for each provided entry.
+          Phone numbers are expected to be in <a href="https://www.twilio.com/docs/glossary/what-e164" target="_blank">E.164</a> format.
+        </p>
+        <form id="form">
+          <div class="form-label-group input-group">
```

Comment:
Why is this form element so large? It's like 3x higher than our other textboxes.

Reply:
I think its the floating labels?

Reply:
I'll take a look.